core/deployment/src/main/java/io/quarkus/deployment/ExtensionLoader.java

```diff
@@ -530,7 +531,7 @@ private static boolean isRecorder(AnnotatedElement element) {
             if (isRecorder) {
                 boolean recorderFound = false;
                 for (Class<?> p : method.getParameterTypes()) {
-                    if (p.isAnnotationPresent(Recorder.class) || p.isAnnotationPresent(Template.class)) {
```

Comment:
How is this change related to suppressing deprecation warnings?

Reply:
It's related because it uses the `isRecorder` method above which has the suppress warning.